src/consts/interfaces.consts.ts

```diff
@@ -93,11 +93,13 @@ export interface IClaimLinkGaslessParams {
 export interface IClaimLinkXChainGaslessParams {
 	APIKey: string
 	baseUrl?: string
+	squidRouterUrl?: string
 	recipientAddress: string
 	link: string
-	destinationChainId: string
+	destinationChainId: number
```

Comment:
ChainId should be a string, cause squid also has chains like cosmos (where the chainId is a string-like)

Reply:
Well yeah, but we don't work with cosmos, and having chain ids everywhere as numbers sounds cleaner to me

Reply:
I disagree, but we can discuss this with @Hugo0 next week.

Reply:
@nebolax @borcherd pls have as string - agreed that number is cleaner for now, but it seems very low cost to have as string, and I'd like to be [CAIP-2](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md) compatible.

Also, Axelar can bridge to cosmos as can Squid ( and potentially 2 other quite big chains in the near term future), which means user should be able to make a withdrawal request to a cosmos chain that we can fulfill